token_manager.py

```
@@ -46,62 +46,99 @@ def set_last_issued_time(self, t):
         except Exception:
             pass
 
-    def can_issue_token(self):
-        """마지막 발급시각 + 24시간 이후에만 True (토큰 파일 없어도 적용)"""
+    def can_issue_token(self, force_if_expired=False):
```

Ruff (0.14.1)

Unused method argument: `force_if_expired`

(ARG002)